backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.permissions import IsAuthenticated
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class UpdateAttendeeTicketError:
+    message: str = ""
```

Comment:
why this empty string?

Reply:
I can add a default message. 
I would like to render the message based on the selected language so on the frontend I just have a generic locale message set when something went wrong